src/react/components/common/imageMap/imageMap.tsx

```diff
@@ -1078,8 +1094,8 @@ export class ImageMap extends React.Component<IImageMapProps> {
         this.initializeDrawnRegionLabelLayer();
         this.initializeDrawnRegionLayer();
         return [this.imageLayer, this.textVectorLayer, this.tableBorderVectorLayer, this.tableIconBorderVectorLayer,
-            this.tableIconVectorLayer, this.checkboxVectorLayer, this.drawRegionVectorLayer, this.labelVectorLayer,
-            this.drawnLabelVectorLayer];
+        this.tableIconVectorLayer, this.checkboxVectorLayer, this.drawRegionVectorLayer, this.labelVectorLayer,
```

Comment:
Is there a auto format tool that you're using? Sometimes there's so many style changes, it's hard to review the actual code for the feature.

Also, some of these style changes seem incorrect., Like here I think there should be an indent.

Reply:
just format the code with the vs code format (Shift+Alt+F)